crates/next-core/src/next_client/context.rs

```diff
@@ -350,6 +350,9 @@ pub async fn get_client_module_options_context(
         enable_postcss_transform,
         side_effect_free_packages: next_config.optimize_package_imports().owned().await?,
         keep_last_successful_parse: next_mode.is_development(),
+        unused_export_removal: *next_config
+            .turbopack_remove_unused_exports(next_mode.is_development())
+            .await?,
```

Comment:
Can we rename `unused_export_removal` to `remove_unused_exports` to not have two names?

Also, you're only enabling it for the browser, not the server: (in `crates/next-core/src/next_server/context.rs`)